Jenkinsfile

```diff
@@ -73,6 +73,8 @@ node('vets-website-linting') {
     try {
       dockerImage.inside(args) {
         sh "cd /application && npm --no-color run test:coverage"
+        sh "cd /application && npm install -g codeclimate-test-reporter"
+        sh "cd /application && CODECLIMATE_REPO_TOKEN=fe4a84c212da79d7bb849d877649138a9ff0dbbef98e7a84881c97e1659a2e24 codeclimate-test-reporter < ./coverage/lcov.info"
```

Comment:
Travis suggests that this token be used publicly, though Code Climate suggests that it be stored as a secret, and that public use could allow others to send coverage reports to our endpoint.

I'm moving this over as is from the Travis implementation, but this should probably be regenerated and transferred in another task. /cc @crwallace